src/server/components/dashboard/listsItems/controllers.ts

```diff
@@ -450,38 +447,3 @@ async function initialiseFormRunnerSession(
   const token = await getInitiateFormRunnerSessionToken(formRunnerNewSessionUrl, formRunnerWebhookData);
   return createFormRunnerEditListItemLink(token);
 }
-
-export async function listItemEditRequestValidation(req: Request, res: Response, next: NextFunction): Promise<void> {
-  logger.info("listItemEditRequestValidation");
-  const userId = req.user?.userData?.id;
-
-  if (userId === undefined) {
-    return res.redirect(authRoutes.logout);
-  }
-
-  const { list, listItem } = res.locals;
-  const listId = list?.id;
-  const listItemId = listItem?.id;
-
-  if (list === undefined) {
```

Comment:
mostly covered by router.params. Also renamed to `requestValidation` since it is already inside the listsItems dir